code/index.html

```diff
@@ -1,22 +1,129 @@
 <!DOCTYPE html>
-<html>
+<html lang="en">
   <head>
     <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
     <title>Project 2: The News Site</title>
     <link rel="stylesheet" href="./style.css" />
-    <!-- dont forget the meta tag for responsiveness -->
   </head>
-  <body>
+   <body>
+ 
+  <!-- HEADER -->
+  
     <header>
-      <!-- Some kind of logo and navbar -->
+      <div class="hamburger">
+        <div></div>
+        <div></div>
+        <div></div>
+      </div>
+      <img src="images/Queens.png" alt="trap queens title">
+      <nav>
+        <a href="#">HOME</a>
+        <a href="#">NEWS</a>
+        <a href="#">ABOUT</a>
+      </nav>
     </header>
 
-    <section class="big-news">
-      <!-- The big news content -->
-    </section>
+     <!-- BIG NEWS -->
 
-    <section class="">
-      <!-- The other news  -->
+    <section class="big-news-container">
+      <section class="container">
+      <img src="images/woman-in-fur-under-neon.jpg" alt="big news">
+      <div>
+        <h2>Smashing news!</h2>
+        <p>
+          What has Tina Trapper been up to? 
+          Find out everything about it. Lorem ipsum dolor sit amet consectetur adipisicing elit. Aliquam quo nihil voluptatem fuga et obcaecati rerum repellat enim repudiandae, vitae numquam
+          Consequuntur et minus esse reiciendis vero quia cumque molestiae?
+        </p>
+      </div>
+    </section>
     </section>
```

Comment:
Nested elements should be indented one level, take a look at the [code guidelines](https://www.notion.so/technigo/Guidelines-for-how-to-write-good-code-69ffb59eac4e4358a7c5752efb873c4e) if you want to recap :)